.github/workflows/reusable/setup.yml

```
@@ -5,10 +5,10 @@ on:
     inputs:
 		node-version:
```

actionlint (1.7.12)

[error] could not parse as YAML: found character that cannot start any token

(syntax-check)